services/object/test/helper.js

```diff
@@ -36,10 +36,16 @@ exports.withBackends = (mock, skipping) => {
     // add the 'test' backend only for testing
     BACKEND_TYPES['test'] = TestBackend;
 
-    // start with an empty set of backends
     await exports.load('cfg');
-    exports.load.cfg('backends', {});
-    exports.load.cfg('backendMap', []);
+    exports.load.cfg('backends', {
+      pub: { backendType: 'test' },
+    });
+    exports.load.cfg('backendMap', [
+      // not anchored, so can appear anywhere in the name
+      { backendId: 'pub', when: { name: { regexp: 'public/.*' } } },
+      // anchored to the beginning of the string
+      { backendId: 'pub', when: { name: { regexp: '^pubdocs/.*' } } },
+    ]);
```

Comment:
Let's use a simpler default that just uses a 'testBackend' backend of type 'test' for all objects.  Otherwise, it's easy to build tests that depend on the details of this default configuration, and then those tests fail if the config is changed.